packages/salesforcedx-vscode-apex-debugger/package.json

```diff
@@ -169,5 +169,10 @@
         "title": "%exception_breakpoint_command_text%"
       }
     ]
+  },
+  "__metadata": {
```

Comment:
@guw What are these for?

Reply:
Hmm, very weird. I haven't noticed. Could that be a leftover from build?

Reply:
@DatGuyJonathan remove it